wallet/src/main/scala/org/bitcoins/wallet/Wallet.scala

```diff
@@ -30,6 +37,41 @@ sealed abstract class Wallet extends LockedWallet with UnlockedWalletApi {
     LockedWallet(nodeApi, chainQueryApi)
   }
 
+  override def createDLCOffer(
+      amount: Bitcoins,
+      oracleInfo: OracleInfo,
+      contractInfo: Vector[Sha256DigestBE],
+      feeRateOpt: Option[SatoshisPerVirtualByte],
+      locktime: UInt32,
+      refundLocktime: UInt32): Future[DLCOffer] = {
+    logger.warn(s"createDLCOffer")
+    val feeRate = feeRateOpt.getOrElse(SatoshisPerVirtualByte.one)
+    for {
+      account <- getDefaultAccount()
+      txBuilder <- fundRawTransactionInternal(
+        destinations = Vector(TransactionOutput(amount, EmptyScriptPubKey)),
+        feeRate = feeRate,
+        fromAccount = account,
+        keyManagerOpt = Some(keyManager)
+      )
+    } yield {
```

Comment:
When running it fails, here. It will output the above "createDLCOffer" and output logs I put both at the end of `getDefaultAccount()` and `fundRawTransactionInternal()` but not "createDLCOffer - DLCOffer"

Reply:
The futures are likely failing some way. It would probably be useful to do something like:
```scala
val resultF = for { ... } yield { ... }
resultF.failed.foreach(err => println(s"err=${err}))
```